src/webots/control/WbLanguageTools.hpp

```diff
@@ -35,6 +35,9 @@ class WbLanguageTools {
 private:
   WbLanguageTools() {}
   ~WbLanguageTools() {}
+  static const QString checkIfPythonCommandExist(const QString &pythonCommand, QProcessEnvironment &env, bool log);
+  // cppcheck-suppress unusedPrivateFunction
+  static QString findWorkingPath(const QString &pythonVersion, QProcessEnvironment &env, bool log);
```

Comment:
That method seems to be specific to python, so I would add "python" in its name:
```suggestion
  static QString findWorkingPythonPath(const QString &pythonVersion, QProcessEnvironment &env, bool log);
```